File: 30 Days of code/src/fr/tenji/js/days4.js
/**
 * Created by nekran on 05/10/16.
 */
// process.stdin.resume();
process.stdin.setEncoding('ascii');
var Person = require('./Person.js');
var input_stdin = '';
var input_stdin_array = [];
var count;
var currentLine = 0;

var readLine = function () {
    return input_stdin_array[currentLine++];
};

var main = function() {
    for (var i = 0; i < input_stdin_array.length; i++) {
        var p = new Person(parseInt(readLine()));
        p.amIOld();
        for (var j = 0; j < 3; j++) {
            p.yearPasses();
        }
        p.amIOld();
        console.log('');
    }
};

process.stdin.on('data', function(data){
    if (count === undefined) {
        count = parseInt(data.trim().replace('\n', ''));
    } else {
        input_stdin += data;
    }
    count--;
    if (count < 0) {
        process.stdin.end();
    }
});

process.stdin.on('close', function() {
    input_stdin_array = input_stdin.split('\n');
    input_stdin_array.pop();
    main();
});

File: 30 Days of code/src/fr/tenji/java/Days4.java
package fr.tenji.java;

import java.util.Scanner;
/**
 * Created by nekran on 05/10/16.
 */
public class Days4 {

    public static void main(String[] args) {

        Scanner scan = new Scanner(System.in);
        int t = scan.nextInt();
        System.out.println("Chosen :" + t);
        Person[] persons = new Person[t];
        for (int i = 0; i < t; i++) {
            int age = scan.nextInt();
            persons[i] = new Person(age);
            persons[i].amIOld();
            for (int j = 0; j < 3; j++) {
                persons[i].yearPasses();
            }
            persons[i].amIOld();
        }
        scan.close();

    }
}
